plugins/modules/aci_config_export_policy.py

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Tim Cragg (@timcragg) <tcragg@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_config_export_policy
+short_description: Manage Configuration Export Policy (config:ExportP)
+description:
+- Manage Configuration Export Policies on Cisco ACI fabrics.
+options:
+  name:
+    description:
+    - The name of the Configuration Export Policy
+    type: str
+  description:
+    description:
+    - Description of the Configuration Export Policy
+    type: str
+  format:
+    description:
+    - Format of the export file. This defaults to json on the APIC.
```

Comment:
Could you consistently add an additional entry for the default in description for all arguments? See example below:

```
  format:
    description:
    - Format of the export file.
    - The APIC defaults to json when unset during creation.